src/Microcks.Testcontainers/MicrocksContainerEnsemble.cs

```diff
@@ -0,0 +1,164 @@
+//
+// Copyright The Microcks Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//
+
+using System.Threading.Tasks;
+using DotNet.Testcontainers.Networks;
+using Microcks.Testcontainers.Connection;
+
+namespace Microcks.Testcontainers;
+
+/// <inheritdoc cref="DockerContainer" />
+public class MicrocksContainerEnsemble : IAsyncDisposable
```

Comment:
Wasn't it possible to follow the `DockerContainer` contract like in `MicrocksContainer.cs`? What is actually an `IAsyncDisposable`? Can you help me understand this providing some pointers?

Reply:
Can be changed later on. I'll mark this one as solved to I can merge it.